lib/venue.rb

```diff
@@ -1,2 +1,30 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+
+  def add_patron()
```

Comment:
In the test, we see we're calling this method like `venue.add_patron('Mike')`. This method is expecting an argument, so when you define it, include an argument:

```ruby
def add_patron(new_patron)
```